10. Databases/3.solution_n3n4

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE test_primary_key (
+    id INTEGER,
+    first_name TEXT NOT NULL,
+    second_name TEXT NOT NULL,
+    age INTEGER,
+    PRIMARY KEY (first_name, second_name)
```

Comment:
👍

Reply:
nit: but if it is required to have pk as id, the unique constraint can be used
`UNIQUE (first_name, last_name)`